### ISSUE TITLE
Bug - Language can not be changed from the Settings > Preferences Menu

### ISSUE THREAD

**Issue by TheWanderingCoder** [kimai/kimai#450]
For both standard users and admin accounts, attempting to change the Language from the Settings > Preferences Menu's Language Option results in no change in display language. The admin accounts' Admin Panel > Advanced > Language Option when selected and Save is pressed does change the display language however.

**Comment by ServiusHack** [comment]
Which version are you using?

Could you check Kimai's logfile and the PHP error log if there is something in there?

If you are not afraid of some code you can help me debug this issue. Open templates/scripts/floaters/preferences.php and add a `debugger;` statement to the success handler, right before `window.location.reload();` is called. When your browser hits this breakpoint check the ajax request if it shows some error and have a look into the database what has been written. That would be a great help.

**Comment by TheWanderingCoder** [comment]
Apologies at taking a while to get back to you. It was a long weekend here in Japan.

I am using the latest release 0.9.2. Although if it would help I can download the files for the 0.9.3 release and try that out.

The logfile (kimai/temporary/logfile.txt) shows the following after trying to change the language settings from the top Settings Icon.
```
[13.01.2015 10:36:11] login: admin as user
[13.01.2015 10:36:52] KSPI axAction (admin): prefs
[13.01.2015 10:36:58] KSPI axAction (admin): 
```

Although I am relatively new to php I gave it a go. The 0.9.2 release has preferences in templates/floaters/preferences.php and I added `debugger`; to the line above the sole `window.location.reload();` call. When I tried to update the language again from the Setting Icon menu, I got no error message /  neither did the language change. Let me know if I did any steps wrong / if there is anything else you would like me to try.

P.S. The code in the preferences file looks like this after I added the `debugger`; line.

```
{literal}    
    <script type="text/javascript"> 
        $(document).ready(function() {


	 var options = { 
		beforeSubmit:  function() { 

                	if ( ($('#password').val() != "" || $('#retypePassword').val() != "")
			      && !validatePassword($('#password').val(),$('#retypePassword').val()))
                	    return false;
					debugger; 
                	window.location.reload();
            	}
	    }; 


...
[Continued as per 0.9.2 release preferences.php file]
```

**Comment by ServiusHack** [comment]
Sorry for the late reply. Please do test 0.9.3 as well. Looking at some older issues there are several reporting this for 0.9.2 and according to the comments this is definitely fixed in 0.9.3. Closing this issue, please reopen if the issue still exists in 0.9.3 for you.
